agent/agent-bootstrap/src/main/java/com/microsoft/applicationinsights/agent/bootstrap/diagnostics/DiagnosticsHelper.java

```diff
@@ -53,11 +53,7 @@ private DiagnosticsHelper() {}
 
   public static void setAgentJarFile(Path agentPath) {
     if (Files.exists(agentPath.resolveSibling("appsvc.codeless"))) {
-      if ("java".equals(System.getenv("FUNCTIONS_WORKER_RUNTIME"))) {
-        rpIntegrationChar = 'f';
-      } else {
-        rpIntegrationChar = 'a';
-      }
+      rpIntegrationChar = 'a';
```

Comment:
I was thinking, maybe we don't want to make this change just yet, in case the new functions deployment model is delayed and/or we roll out another update to App Services before the new functions deployment model is launched